client/src/components/Header.js

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+
+
+export default function Header({ open, setOpen, headline }) {
+
+    return (
+        <Wrapper open={open}>
+            <a onClick={() => setOpen(!open)}>
+                <svg xmlns="http://www.w3.org/2000/svg" width="28.25" height="23.101" viewBox="0 0 28.25 23.101">
+                    <g id="Icon_feather-menu" data-name="Icon feather-menu" transform="translate(-4 -8.5)">
+                        <path id="Pfad_19" data-name="Pfad 19" d="M4.5,18H31.75" transform="translate(0 2.051)" fill="none" stroke="#fff" strokeLinecap="round" strokeLinejoin="round" strokeWidth="1" />
+                        <path id="Pfad_20" data-name="Pfad 20" d="M4.5,9H31.75" fill="none" stroke="#fff" stroke-linecap="round" strokeLinejoin="round" strokeWidth="1" />
+                        <path id="Pfad_21" data-name="Pfad 21" d="M4.5,27H31.75" transform="translate(0 4.101)" fill="none" stroke="#fff" strokeLinecap="round" strokeLinejoin="round" strokeWidth="1" />
+                    </g>
+                </svg>
```

Comment:
In React it is pretty easy to import the SVGs so you don't need to add the whole SVG code but can only import the Icon from an `assets`folder. 

The syntax would be like 
`import { ReactComponent as YourIcon } from "./icon.svg"`

and then you can simply add  the icon to your component: 
`<a onClick={() => setOpen(!open) )}>
 <MyIcon />
</a >
`

You can read more for example [in this post](https://betterprogramming.pub/react-best-way-of-importing-svg-the-how-and-why-f7c968272dd9) --> Scroll down to **Import SVG as a React Component (Inline SVG)**